src/explore-education-statistics-common/src/components/AccordionSection.tsx

```diff
@@ -55,14 +55,17 @@ const AccordionSection = ({
   heading,
   headingTag = 'h2',
   id = 'accordionSection',
-  anchorLinkIdPrefix = 'section',
+  anchorLinkIdPrefix,
```

Comment:
Can we not set a default for this? We need to be able to prevent invalid anchor ids from headings that start with numbers